sdk/lib/opentelemetry/sdk/internal.rb

```diff
@@ -16,6 +16,16 @@ def boolean?(value)
         value.is_a?(TrueClass) || value.is_a?(FalseClass)
       end
 
+      def to_boolean(value)
+        return nil if value.nil?
+        return value if boolean?(value)
+
+        case value.to_s
+        when /\A(true|t|yes|y|1)\z/i then true
+        when /\A(false|f|no|n|0)\z/i then false
+        end
```

Comment:
This flexibility is overkill IMO. Simply supporting `true` and `false` is sufficient for boolean env vars. See the [Insecure option for OTLP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/exporter.md#configuration-options) for example.

Reply:
ok